app/lib/search_query_builder.rb

```diff
@@ -116,12 +122,28 @@ def default_order
     finder_content_item['details']['default_order'] || "-public_timestamp"
   end
 
-  def filter_query
-    filter_params
-      .merge(base_filter)
-      .reduce({}) { |query, (k, v)|
-        query.merge("filter_#{k}" => v)
-      }
+  def base_filter_query
+    @base_filter_query ||= base_filter.each_with_object({}) do |(k, v), query|
+      query["filter_#{k}"] = v
+    end
```

Comment:
Should this include the `and_filter_query`?

Reply:
It shouldn't. We've changed the way the finder works so that the `or` facets are now completely independent of the `and` facets - after much discussion we've concluded this logic makes the most sense. Therefore, this `base_filter_query` (which is the base for both the `and` and the `or` queries) needs to only contain the finder base filter and nothing else.

Reply:
We also realised that if the `and` query was to be included as the base query for each of the `or` queries, ticking any `or` facets wouldn't surface any more results as they would have already been included in the base `and` query which would've had a wider criteria than the `and` + `or` query.

Reply:
Ok, so the desired logic now is: all AND facets hold, or at least one OR facet holds?

Reply:
Yeah, that's correct. It's basically: `(and_facet_1 AND and_facet_2) OR or_facet_1 OR or_facet_2 OR ...`

Reply:
Maybe instead of the "X results for businesses which Yes" we should instead say "X results for (facet_1 && ...) || facet_3 || ..."!